### ISSUE TITLE
Very large picture file decoding error

### ISSUE THREAD

**Issue by zern** [lovasoa/dezoomify-rs#9]
In general, this program works very well.
However, a small number of images are saved as JPG, limited by the jpg file format (maximum image size of 65,535 x 65,535 pixels).
I changed the format to the output png/tif file and saved it successfully, but couldn't open it, prompting the decoding error.
Test links:
https://artsandculture.google.com/asset/reproduction-of-admonitions-scroll/mgFUbzN9ERoGWg
How can you fix this problem? Thank you

**Comment by lovasoa** [comment]
Hello, can you please provide the PNG file that fails to open?

**Comment by zern** [comment]
Thank you for your help.
This file is very large. I uploaded it to google drive and you can download it
https://drive.google.com/open?id=1BL1W1pjUqL2bK_H8LIfwIRTogd75h4jX

**Comment by MrChrisWin** [comment]
Does this mean dezoomify can output to format different than JPG?

**Comment by lovasoa** [comment]
Yes! Read the readme at the root of this project for the list of options dezoomify has. 

You can simply call :

```
dezoomify-rs "https://yoursite.com" myfilename.png
```

And the result will be saved as a png file.

**Comment by MrChrisWin** [comment]
Unless I miss something, the README file only mentions Output file and does not specify the supportable file format.  I can edit the README to be more specific.
So far we know it can support JPG (length or width cannot exceed 65,000 pixels) and PNG (limited by RAM). What about TIFF support?

**Comment by lovasoa** [comment]
Yes, improvements to the readme and the documentation are welcome 👍  
The formats we support are the ones supported by the [image-rs library](https://github.com/image-rs/). They are listed [here](https://github.com/image-rs/image#2-supported-image-formats)

**Comment by AlexanderDeLargeBrotherSir** [comment]
@zern, You need a good pc with good hardware to open such a big file in photoshop. 
Close this right now
